sample.env

```diff
@@ -0,0 +1,16 @@
+# copy this file to .env and edit to set custom values.
+
+# Port you wish to access Django's dev server on.
+PORT=8001 
```

Comment:
Probably start with this one commented out, so someone who does copy it to .env per the README but makes no changes actually gets it running on 8000, which is the normal expectation for a lot of people.